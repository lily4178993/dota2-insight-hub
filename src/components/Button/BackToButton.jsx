/* eslint-disable jsx-a11y/control-has-associated-label */
import React from 'react';
import './backToButton.css';

function BackToButton() {
  return (
    <div className="backButton">
      <button type="button" title="Go back">
        <i className="bx bx-chevron-left" />
      </button>
    </div>
  );
}

export default BackToButton;
