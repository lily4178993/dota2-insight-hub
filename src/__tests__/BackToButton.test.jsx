import React from 'react';
import renderer from 'react-test-renderer';
import BackToButton from '../components/Button/BackToButton';

test('BackToButton component matches snapshot', () => {
  const component = renderer.create(<BackToButton />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
