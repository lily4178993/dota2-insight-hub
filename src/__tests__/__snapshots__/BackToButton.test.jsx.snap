// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackToButton component matches snapshot 1`] = `
<div
  className="backButton"
>
  <button
    type="button"
  >
    <i
      className="bx bx-chevron-left"
    />
  </button>
</div>
`;
